scripts/DDL.sql

```diff
@@ -68,6 +68,9 @@ CREATE TABLE jobs.advertisements(
 	contact text,
 	base_location_id int REFERENCES jobs.base_locations(id) ON DELETE CASCADE,
 	is_close boolean DEFAULT false,
+	advertisment_date bigint,
```

Comment:
why not date? you need a timestamp?

Reply:
Decided on number to save on conversions, how would you save it as Date in db and initialize with "now"?

Reply:
timestamp is better here IMO

Reply:
change it to timestamp